### PR TITLE
Added support for PanelOne from T3P3.

### PR DESCRIPTION
**Pull request by maverikou**

### CODE REVIEW

File: Marlin/Configuration.h [hunk 1]
```diff
@@ -639,6 +643,10 @@ const bool Z_MAX_ENDSTOP_INVERTING = true; // set to true to invert the logic of
  #define ENCODER_STEPS_PER_MENU_ITEM 1
 #endif
 
+#if defined (PANEL_ONE)
+ #define SDSUPPORT
+ #define ULTIMAKERCONTROLLER
+#endif
```
Comment (thinkyhead) [inline Marlin/Configuration.h:650]: Sweet! If you wouldn't mind adding these lines to the other `Configuration.h` files (in example_configurations) we can merge this forthwith!

Comment (maverikou) [inline Marlin/Configuration.h:650]: Sure

**Comment by maverikou** [comment]
What else is required, @thinkyhead ?

**Comment by thinkyhead** [comment]
Looks good!

**Pull request merged**